RealmSwift/Error.swift

```diff
@@ -274,14 +219,4 @@ public func == (lhs: ErrorType, rhs: ErrorType) -> Bool { // swiftlint:disable:t
         && lhs._domain == rhs._domain
 }
 
-// MARK: Pattern Matching
-
-/**
- Pattern matching matching for `Realm.Error`, so that the instances can be used with Swift's
- `do { ... } catch { ... }` syntax.
-*/
-public func ~= (lhs: Error, rhs: ErrorType) -> Bool { // swiftlint:disable:this valid_docs
```

Comment:
Actually, I mistook the lines to delete. The code I really wanted to delete is Swift 3's code.